mackup/applications/rtx.cfg

```diff
@@ -0,0 +1,14 @@
+[application]
+name = rtx
+
+[configuration_files]
+.config/rtx
+.rtx.toml
+.tool-versions
```

Comment:
This file and the ones below seem totally generic, are those really owned by rtx? Why does it have both a `.config` folder and lots of files in the user's home?

Reply:
These files relate to [asdf](https://github.com/lra/mackup/blob/64d6687/mackup/applications/asdf.cfg) and rtx, which is an [asdf rust clone](https://github.com/lra/mackup/blob/64d6687/mackup/applications/asdf.cfg).

Reply:
Makes sense, thanks.